app/src/main/java/org/mozilla/fenix/search/telemetry/SearchProviderModel.kt

```diff
@@ -39,7 +39,7 @@ data class SearchProviderModel(
      * Checks if any of the given URLs represent an ad from the search engine.
      * Used to check if a clicked link was for an ad.
      */
-    fun containsAds(urlList: List<String>) = urlList.any { url -> isAd(url) }
+    fun hasAdClicks(urlList: List<String>) = urlList.any { url -> isAd(url) }
```

Comment:
This was a simple refactor of the naming. I think this is closer to our understanding of the search telemetry terminologies.

Reply:
I find the name a little strange - you're checking a url list for any of the urls being ad links, but then you're calling it a "click".

The function appears to be used for both when a SERP page might contain ad links and when a user clicks a link.

I think `containsAds` was probably more appropriate, or maybe even better would be `containsAdLinks` seeing as you're testing a list for ad links.